### ISSUE TITLE
Namespace' object has no attribute 'c11

### ISSUE THREAD

**Issue by masud407** [PALab/RUS#17]
When I run the RUS main file on pyhton, I get the following attribute error:

**AttributeError: 'Namespace' object has no attribute 'c11'**

Any suggestions to fix it?

**Comment by jamishepherd** [comment]
Hi there, this could be related to the Python version used.  We have no errors when using Python 3.7.  We recommend updating your Python version and trying again.

**Comment by masud407** [comment]
Dr. Shepherd, thanks a lot for your response. Yes, I am actually using Python 3.8. 
The error is specifically at line 234 for the rus_perser.py file (post-processing function).

**Comment by kaspervanwijk** [comment]
If you send us the commands you run, and the version of the RUS code,
python and OS, we can try to reproduce your result.

Cheers,
Kasper

**Comment by masud407** [comment]
Dr. Kasper,

Thanks for your response. I used the code from the following link:
https://github.com/PALab/RUS/tree/master/python

I just run the rus.py file which contains the following lines:
###
import rus_parser as parser
import rus_forward as forward
import rus_inverse as inverse

if __name__ == "__main__":

    args = parser.start()

    if args.subcommand == 'forward':
        forward.start(args)

    if args.subcommand == 'inverse':
        inverse.start(args)
###
After that, I am getting the following error:

###
  **File "*****\RUS-master\python\rus_parser.py", line 234, in post_process
    if args.c11: args.a['c11'] = args.c11

AttributeError: 'Namespace' object has no attribute 'c11'**
###

**Comment by kaspervanwijk** [comment]
Rus.py requires arguments, among them, c11.

**Comment by masud407** [comment]
Dr. Kasper,

Sorry, I didn't understand what you meant by "Rus.py requires arguments, among them, c11". Will it be possible to elaborate it?

**Comment by kaspervanwijk** [comment]
Yes, I can: you can learn how to use the forward code by typing:

python rus.py forward --help

ngā mihi nui,
Kasper van Wijk
Department of physics
University of Auckland
https://www.physics.auckland.ac.nz/people/kvan637
